spec/workflow_service_spec.rb

```diff
@@ -464,6 +464,31 @@
     end
   end
 
+  describe '#count_objects_in_step' do
+    before(:all) do
+      @workflow   = 'sdrIngestWF'
+      @step       = 'start-ingest'
+      @type       = 'waiting'
+      @repository = 'sdr'
```

Comment:
@tallenaz not a big deal, but these read like they should be `let` statements to me